c/tskit/genotypes.h

```diff
@@ -33,7 +33,10 @@ extern "C" {
 #include <tskit/trees.h>
 
 #define TSK_16_BIT_GENOTYPES (1 << 0)
-#define TSK_IMPUTE_MISSING_DATA (1 << 1)
+#define TSK_ISOLATED_NOT_MISSING (1 << 1)
+
+/* This flag was deprected in 0.3.0 and replaced with TSK_ISOLATED_NOT_MISSING */
+#define TSK_IMPUTE_MISSING_DATA TSK_ISOLATED_NOT_MISSING
```

Comment:
Just nuke it - this is why we're still on 0.99, we reserve the right to break people's code.

Reply:
Unless you object @petrelharp?

Reply:
no problems here

Reply:
Removed in https://github.com/tskit-dev/tskit/pull/794/commits/1cc1e9eb659bdc71f83d74d262a8a488f0875dda

Reply:
In retrospect was silly to refer to the python version number in the C....